src/System.Private.ServiceModel/tests/Scenarios/Contract/XmlSerializer/XmlSerializerFormatTest.cs

```diff
@@ -69,4 +69,71 @@ public static void XmlSerializerFormatAttribute_SupportFaults()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
+
+    [WcfFact]
+    [OuterLoop]
+    public static void XmlSFAttributeWsdlTest()
+    {
+        RunVariation(Endpoints.BasciHttpRpcEncSingleNs_Address);
+        RunVariation(Endpoints.BasicHttpRpcLitSingleNs_Address);
+        RunVariation(Endpoints.BasicHttpDocLitSingleNs_Address);
+        RunVariation(Endpoints.BasicHttpRpcEncMultiNs_Address, true);
+        RunVariation(Endpoints.BasicHttpRpcLitMultiNs_Address, true);
+        RunVariation(Endpoints.BasicHttpDocLitMultiNs_Address, true);
+    }
+
+    private static void RunVariation(string serviceAddress, bool isMultiNs = false)
+    {
+        BasicHttpBinding binding = null;
+        EndpointAddress endpointAddress = null;
+        ChannelFactory<ICalculator> factory1 = null;
+        ChannelFactory<IHelloWorld> factory2 = null;
+        ICalculator serviceProxy1 = null;
+        IHelloWorld serviceProxy2 = null;
+
+        // *** SETUP *** \\
+        binding = new BasicHttpBinding();
+        endpointAddress = new EndpointAddress(serviceAddress);
+        factory1 = new ChannelFactory<ICalculator>(binding, endpointAddress);
+        serviceProxy1 = factory1.CreateChannel();
+        if (isMultiNs)
+        {
+            factory2 = new ChannelFactory<IHelloWorld>(binding, endpointAddress);
+            serviceProxy2 = factory2.CreateChannel();
+        }
+
+        // *** EXECUTE Variation *** \\
+        try
+        {
+            var intParam = new IntParams() { p1 = 5, p2 = 10 };
+            var floatParam = new FloatParams() { p1 = 5.0f, p2 = 10.0f };
+            var byteParam = new ByteParams() { p1 = 5, p2 = 10 };
+
+            Assert.Equal(3, serviceProxy1.Sum2(1, 2));
+            Assert.Equal(intParam.p1 + intParam.p2, serviceProxy1.Sum(intParam));
+            Assert.Equal(string.Format("{0}{1}", intParam.p1, intParam.p2), serviceProxy1.Concatenate(intParam));
+            Assert.Equal((float)(floatParam.p1 / floatParam.p2), serviceProxy1.Divide(floatParam));
+            Assert.Equal((new byte[] { byteParam.p1, byteParam.p2 }), serviceProxy1.CreateSet(byteParam));
+            Assert.Equal(DateTime.Now.Date, serviceProxy1.GetCurrentDateTime().Date);
+            serviceProxy1.DoSomething(intParam);
```

Comment:
Per my comments on `DoSomething` below, we need to do verification on this.

Reply:
Updated, replaced with SetIntParamsProperty(IntPrams) and GetIntParamsProperty().